test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -0,0 +1,181 @@
+const mockLoggerDebugRequest = logger.debugRequest as jest.Mock;
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockCompanyAuthenticationMiddleware = companyAuthentication as jest.Mock;
+mockCompanyAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockTransactionService = postTransaction as jest.Mock;
+mockTransactionService.mockReturnValue( TRANSACTION_ID );
+
+const mockOverseasEntity = updateOverseasEntity as jest.Mock;
+mockOverseasEntity.mockReturnValue( OVERSEAS_ENTITY_ID );
+
+const mockCloseTransaction = closeTransaction as jest.Mock;
+mockCloseTransaction.mockReturnValue( TRANSACTION_CLOSED_RESPONSE );
+
+const mockPaymentsSession = startPaymentsSession as jest.Mock;
+mockPaymentsSession.mockReturnValue( "CONFIRMATION_URL" );
+
+describe("Update review overseas entity information controller tests", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockGetApplicationData.mockReturnValue({
+      ...APPLICATION_DATA_CH_REF_UPDATE_MOCK,
+    });
+  });
+
+  describe("GET tests", () => {
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page`, async () => {
+      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_UPDATE_BO_MOCK);
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_PAGE_TITLE);
+      expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
+      expect(resp.text).toContain(CONTINUE_BUTTON_TEXT);
+    });
+
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page with contact details section with (ceased) existing BO`, async () => {
+      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_CH_REF_UPDATE_MOCK);
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_PAGE_TITLE);
+      expect(resp.text).toContain(UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL);
+      expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENT);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENTS_CEASED_TITLE);
+      expect(resp.text).toContain(UPDATE_PRESENTER_CHANGE_FULL_NAME);
+      expect(resp.text).toContain(UPDATE_PRESENTER_CHANGE_EMAIL);
```

Comment:
I don't think we're confirming the 'Beneficial owners' heading exists for this test or anything within that section

Reply:
No 'Beneficial owners' heading exists does not exist in the contact details section. Test renamed to reflect contact details and beneficial owner statements